Packages/Tracking/Examples~/Hands Examples/Scripts/CycleHandPairs.cs

```diff
@@ -13,40 +13,47 @@ namespace Leap.Unity.HandsModule.Examples
 {
     public class CycleHandPairs : MonoBehaviour
     {
-        public List<GameObject> handList;
+        [SerializeField]
+        private HandModelManager handModelManager;
         private int currentHandID;
 
         // Use this for initialization
         void Start()
         {
+            if (handModelManager == null)
+            {
+                handModelManager = FindObjectOfType<HandModelManager>();
+            }
             currentHandID = 0;
+            handModelManager.EnableHandModelPair(currentHandID, true);
         }
 
         // Update is called once per frame
         void Update()
         {
             if (Input.GetKeyUp(KeyCode.RightArrow))
             {
-                currentHandID++;
-                if (currentHandID > handList.Count - 1) currentHandID = 0;
+                NextHandSet();
             }
 
             if (Input.GetKeyUp(KeyCode.LeftArrow))
             {
-                currentHandID--;
-                if (currentHandID < 0) currentHandID = handList.Count - 1;
+                PreviousHandSet();
             }
+        }
 
-            SortHands();
+        private void NextHandSet()
+        {
+            currentHandID++;
+            if (currentHandID > handModelManager.handModelPairs.Count - 1) currentHandID = 0;
+            handModelManager.EnableHandModelPair(currentHandID, true);
```

Comment:
The assumption here is when you enable a pair the previously enabled pair is disabled? If so, could the naming be clearer?

Reply:
The method has an optional param which defaults to true which disables all other active registered hand models.
I've explicitly set to true here to call it out to developers looking at the sample code. 

I'm happy to rename the method or split it out into two methods (EnableHandModelPair & EnableHandModelPairAndDisableOtherActiveHandModels) if you'd prefer?

Reply:
Or could be handModelManager.EnableHandModelPair(currentHandID, disableOtherHandsOfSameChirality:true);

Reply:
ooh i didnt know named parameters were a feature of c#! will amend.

Reply:
done! Are you happy with that now?

Reply:
Yes